ccl/changefeedccl: TestChangefeedKafkaMessageTooLarge failed

ccl/changefeedccl.TestChangefeedKafkaMessageTooLarge [failed](https://mesolite.cluster.engflow.com/invocations/default/0f817728-2248-4360-8402-bf6bbb54cc50?testReportRun=2&testReportShard=16&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [a2a990025fe761b9da93c1c68c223ff48a420f04](https://github.com/cockroachdb/cockroach/commits/a2a990025fe761b9da93c1c68c223ff48a420f04):


```
=== RUN   TestChangefeedKafkaMessageTooLarge
    test_log_scope.go:171: test logs captured to: outputs.zip/logTestChangefeedKafkaMessageTooLarge3661674579
    test_log_scope.go:82: use -show-logs to present logs inline
    changefeed_test.go:11180: -- test log scope end --
test logs left over in: outputs.zip/logTestChangefeedKafkaMessageTooLarge3661674579
--- FAIL: TestChangefeedKafkaMessageTooLarge (44.94s)
=== RUN   TestChangefeedKafkaMessageTooLarge/kafka
    helpers_test.go:1391: making server as secondary tenant
    helpers_test.go:1470: making kafka feed factory
    testfeed_test.go:341: creating external connection
    testfeed_test.go:344: ran create external connection
    testfeed_test.go:341: creating external connection
    testfeed_test.go:344: ran create external connection
    testfeed_test.go:341: creating external connection
    testfeed_test.go:344: ran create external connection
    testfeed_test.go:341: creating external connection
    testfeed_test.go:344: ran create external connection
    testfeed_test.go:341: creating external connection
    testfeed_test.go:344: ran create external connection
    testfeed_test.go:341: creating external connection
    testfeed_test.go:344: ran create external connection
--- FAIL: TestChangefeedKafkaMessageTooLarge/kafka (44.93s)
=== RUN   TestChangefeedKafkaMessageTooLarge/kafka/succeed_against_a_large_backfill
    changefeed_test.go:11107: 
        	Error Trace:	pkg/ccl/changefeedccl/helpers_test.go:279
        	            				pkg/ccl/changefeedccl/helpers_test.go:498
        	            				pkg/ccl/changefeedccl/changefeed_test.go:11107
        	Error:      	Received unexpected error:
        	            	operation "kafka.Next-1115005548675825665" timed out after 30s (given timeout 30s): context deadline exceeded
        	            	(1) operation "kafka.Next-1115005548675825665" timed out after 30s (given timeout 30s)
        	            	Wraps: (2) context deadline exceeded
        	            	Error types: (1) *timeutil.TimeoutError (2) context.deadlineExceededError
        	Test:       	TestChangefeedKafkaMessageTooLarge/kafka/succeed_against_a_large_backfill
--- FAIL: TestChangefeedKafkaMessageTooLarge/kafka/succeed_against_a_large_backfill (32.48s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=2</code>
 - <code>shard=16</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedKafkaMessageTooLarge.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-55381

--- a/pkg/ccl/changefeedccl/changefeed_test.go
+++ b/pkg/ccl/changefeedccl/changefeed_test.go
@@ -11041,6 +11041,8 @@ func TestChangefeedKafkaMessageTooLarge(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
+	skip.UnderDuress(t, "prone to overload")
+
 	testFn := func(t *testing.T, s TestServer, f cdctest.TestFeedFactory) {
 		if KafkaV2Enabled.Get(&s.Server.ClusterSettings().SV) {
 			// This is already covered for the v2 sink in another test: TestKafkaSinkClientV2_Resize

